app-shared/src/androidJvmMain/kotlin/io/github/droidkaigi/confsched/extension/MouseNavigationSupportForDesktop.kt

```diff
@@ -0,0 +1,31 @@
+package io.github.droidkaigi.confsched.extension
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.navigation3.runtime.NavKey
+import io.github.droidkaigi.confsched.component.rememberOneStepForwardControllerForDesktop
+import io.github.droidkaigi.confsched.navkey.TimetableNavKey
+
+@Composable
+@OptIn(ExperimentalComposeUiApi::class)
+internal fun Modifier.mouseNavigationSupportForDesktop(
+    backStack: SnapshotStateList<NavKey>,
```

Comment:
Since we're dealing with NavKey, we couldn't use the expect/actual method, so this is the form it took.

Reply:
You can define the expect function inside the androidJvmMain source set. This way, you can completely eliminate the iOS actual function and keep the codebase clean!

```kotlin
// androidJvmMain
@Composable
expect fun Modifier.mouseNavigationSupportForDesktop(
    backStack: SnapshotStateList<NavKey>,
): Modifier

// androidMain
actual fun Modifier.mouseNavigationSupportForDesktop(
    backStack: SnapshotStateList<NavKey>,
): Modifier {
    // NOOP
}

// jvmMain
actual fun Modifier.mouseNavigationSupportForDesktop(
    backStack: SnapshotStateList<NavKey>,
): Modifier {
    // Desktop only implementation
}
```